spikeinterface/core/unitsaggregationsorting.py

```diff
@@ -0,0 +1,93 @@
+from typing import List, Union
+
+import numpy as np
+
+from .basesorting import BaseSorting, BaseSortingSegment
+
+
+class UnitsAggregationSorting(BaseSorting):
+    """
+    Class that handles aggregating units from different sortings, e.g. from different channel groups.
+
+    Do not use this class directly but use `si.aggregate_units(...)`
+
+    """
+    def __init__(self, sorting_list):
```

Comment:
The unit_map shoudl be an optional kwargs.

Reply:
to rename units?

Reply:
added optional renaming